encrypted-media-respec.html

```diff
@@ -600,6 +600,13 @@
             multiple independent profiles for a single account.
           </p>
         </dd>
+        <dt id="valid-media-mime-type">Valid Media MIME Type</dt>
+        <dd>
+          <p>
+            A valid media MIME type is a media <a def-id="mime-types">MIME type</a> that is also a <a def-id="valid-mime-type"></a> [[!HTML51]].
+            When a MIME type includes parameters, such as `"codecs"` [[!RFC6381]], such parameters MUST also be valid per the relevant specification.
+            When used with the features defined in this specification, MIME type strings SHOULD explicitly specify codecs and codec constraints (e.g., per [[!RFC6381]]) unless these are normatively implied by the container.</p>
```

Comment:
This last sentence is not strictly part of the definition. It is more a reference to the fact that the type will be rejected, but not because it is _invalid_.

Thus, I suggest making it a separate paragraph. WDYT?